rackcity/views/rack_views.py

```diff
@@ -26,3 +30,66 @@ def rack_detail(request, pk):
     if request.method == 'GET':
         serializer = RackSerializer(rack)
         return JsonResponse(serializer.data)
+
+
+@api_view(['POST'])
+@permission_classes([IsAuthenticated])
+def rack_get(request):
+    """
+    List all racks within specified range.
+    """
+    range_serializer = RackRangeSerializer(data=request.data)
+    if range_serializer.is_valid():
+        try:
+            range_serializer.validate(request.data)
+        except Exception as error:
+            return JsonResponse(
+                {"failure_message": str(error)},
+                status=HTTPStatus.BAD_REQUEST
+            )
+    else:
+        return JsonResponse(
+            {"failure_message": "invalid keys or values"},
+            status=HTTPStatus.BAD_REQUEST,
+        )
+
+    racks = Rack.objects.filter(
+        rack_num__range=range_serializer.get_number_range(request.data),
+        row_letter__range=range_serializer.get_row_range(request.data),
```

Comment:
The `__range` filter is inclusive on both max and min

Reply:
Worth including as an inline comment?

Reply:
done